pkg/postage/postagecontract/contract.go

```diff
@@ -114,11 +114,16 @@ func (c *postageContract) sendApproveTransaction(ctx context.Context, amount *bi
 }
 
 func (c *postageContract) sendTransaction(ctx context.Context, callData []byte, desc string) (*types.Receipt, error) {
+	gasLimit := sctx.GetGasLimit(ctx)
+	if gasLimit == 0 {
+		// if gas limit is not set, use the default limit.
+		gasLimit = 9_000_000
+	}
```

Comment:
would it be beneficial if new method is added to replace this usage with single call  `sctx.GetGasLimitDefaultTo(ctx, 9_000_000)`